typings/node-pty.d.ts

```diff
@@ -77,5 +79,10 @@ declare module 'node-pty' {
      * @throws Will throw when signal is used on Windows. 
      */
     kill(signal?: string): void;
+
+    /**
+     * Return the socket this instance is using to communicate with the internal pty. Use at your own risk!
```

Comment:
I don't really want to support something like this without a compelling use case. People who want to use at their own risk  may as well just access `Terminal._socket` directly?